src/config.c

```
@@ -3146,6 +3147,15 @@ static int applyClientMaxMemoryUsage(const char **err) {
     return 1;
 }
 
+static int applyClusterSlotMigrationLogConfig(const char **err) {
+    if (server.cluster_enabled) {
```

We typically just silently accept configs that don't apply, that way devops folks can have the same config regardless of cluster configuration.

---

Sounds good, changed